modules/nw-configuring-lb-allowed-source-ranges-migration.adoc

```diff
@@ -0,0 +1,66 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer-allowed-source-ranges.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-lb-allowed-source-ranges-migration_{context}"]
+= Migrating to Load Balancer allowed source ranges
+
+If you have already set the annotation `service.beta.kubernetes.io/load-balancer-source-ranges`, you can migrate to Load Balancer allowed source ranges. When you set the `AllowedSourceRanges`, the Ingress Controller sets the `Spec.LoadBalancerSourceRanges` based on the `AllowedSourceRanges` and unsets the `service.beta.kubernetes.io/load-balancer-source-ranges` annotation.
+
+[NOTE]
+====
+If you have already set the `.Spec.LoadBalancerSourceRanges` field or the Load Balancer service anotation `service.beta.kubernetes.io/load-balancer-source-ranges` in a previous version of {product-title}, Ingress Controller starts reporting `Progressing=True` after an upgrade. To fix this, set `AllowedSourceRanges` that overwrites `.Spec.LoadBalancerSourceRanges` field and clears `service.beta.kubernetes.io/load-balancer-source-ranges` annotation. Ingress Controller starts reporting `Progressing=False` again.
+====
+
+.Prerequisites
+
+* You have set the `service.beta.kubernetes.io/load-balancer-source-ranges` annotation.
+
+.Procedure
+
+. Ensure that the `service.beta.kubernetes.io/load-balancer-source-ranges` is set:
++
+[source,terminal]
+----
+$ oc get svc router-default -n openshift-ingress -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.kubernetes.io/load-balancer-source-ranges: 192.168.0.1/32
+----
+
+. Ensure that the `.Spec.LoadBalancerSourceRanges` is unset:
```

Comment:
Should this capitalization match the below yaml (so, lowercase initial "s" in "spec" and "l" in "loadBalancerSourceRanges")?

Reply:
Yes. Addressed